leo_docking/leo_docking/utils.py

```diff
@@ -298,7 +308,7 @@ def calculate_threshold_distances(board: BoardPose) -> Tuple[float, float]:
     x_cross = (perpend_coeff_b - coeff_b) / (coeff_a - perpend_coeff_a)
     y_cross = coeff_a * x_cross + coeff_b
 
-    y_dist = math.sqrt(x_cross ** 2 + y_cross ** 2)
+    y_dist = math.sqrt(x_cross**2 + y_cross**2)
 
     x_dist = math.sqrt((x_cross - position.x()) ** 2 + (y_cross - position.y()) ** 2)
```

Comment:
Consistency with the whitespace on the math operators. Maybe add the spaces back to the y_dist line for the ** operator

Reply:
This might be a black thing. I have a vague memory of seeing it handle things differently with the powers when there are parentheses ahead of it vs. when there are not. :thinking: